docs/package.json

```diff
@@ -114,8 +114,9 @@
     "redux-logger": "^3.0.6",
     "rimraf": "^3.0.0",
     "styled-components": "^5.0.0",
-    "stylis-plugin-rtl": "^1.0.0",
     "stylis": "^3.5.4",
+    "stylis-plugin-rtl": "^1.0.0",
+    "theme-ui": "^0.3.1",
```

Comment:
Should the dependency be listed inside the main package? It doesn't seem related to the documentation.

Reply:
@eps1lon do you have a preference around how much scoped dependencies are? If we want it to be scoped, we would need a package.json file for the benchmark.